backend/controllers/getAuthRoles.js

```diff
@@ -0,0 +1,17 @@
+const { getGoogleGroup } = require('./googlegroup.controller')
+
+export const getAuthRoles = async (identifier) => {
```

Comment:
move auth overrides to here, make sure all `isAdmin`, `isTeamCaptain`, `isDirector` references in the backend come through the getAuthRoles hook